service/model/settings.go

```diff
@@ -0,0 +1,122 @@
+package model
+
+import (
+	"errors"
+)
+
+var (
+	errOrganizationNameRequired  = errors.New("organization name is required")
+	errDefaultReleaseMsgRequired = errors.New("default release message is required")
+	errSlackMissingToken         = errors.New("token is required to enable slack integration")
+	errGithubMissingToken        = errors.New("token is required to enable github integration")
+)
+
+type Settings struct {
+	OrganizationName      string
+	DefaultReleaseMessage string
+	Slack                 SlackSettings
+	Github                GithubSettings
+}
+
+type SettingsUpdate struct {
```

Comment:
Ok, this one I am going to update in this PR. And then I will refactor the rest in the separate PR :)

What name would you use for the struct in the repository layer? Would you choose UpdateSettingsInput as well?

Reply:
On the repository layer, since you are upserting and always using the full type itself, there is no need to have a `UpdateSettingsInput` type, and that's totally fine.